### DISCUSSION TITLE
Issue with copy and paste images in reporting

### DISCUSSION THREAD

**Discussion by unfrs** [Syslifters/sysreptor#366] [Q&A]
Hey guys, I have a problem with copying and pasting images inside my report.

[https://docs.sysreptor.com/show-and-tell/2023/05/30/copy-and-paste-image-to-pentest-report/](url)
I did the same in my report. 

When I'm reporting everything works fine.
<img width="1039" alt="Screenshot 1446-06-14 at 6 10 01 PM" src="https://github.com/user-attachments/assets/19173a53-ec69-4ba3-966a-427c161bc4ed" />

But the image is not saved in the db and it displays like this when I publish the report.
<img width="526" alt="Screenshot 1446-06-14 at 6 08 13 PM" src="https://github.com/user-attachments/assets/b89bd822-c713-44bd-a664-5ab9388f0545" />

**Comment by aronmolnar** [comment]
Duplicate #367
